communitymanagement/src/main/java/communitymanagement/dao/TicketCommentDao.java

```diff
@@ -0,0 +1,52 @@
+package communitymanagement.dao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
```

Comment:
delete this empty line -- recommend to use the "format" function in the eclipse :)